.github/ISSUE_TEMPLATE/feature.yml

```diff
@@ -0,0 +1,28 @@
+name: 💡 Feature Request
+description: Tell us about a problem our software could solve but doesn't.
```

Comment:
```suggestion
description: Tell us about a problem our software could solve but doesn't, yet.
```

Reply:
Love the optimism but don't want to overcommit our PM/POs. 😁